TASVideos/wwwroot/js/site.js

```diff
@@ -73,3 +73,8 @@ if (localStorage.getItem("style-dark") !== null) {
 		forceDarkMode();
 	}
 }
+
+if (location.hash != null && location.hash != "") {
```

Comment:
if (location.hash) is cleaner here

Reply:
I do not know the policy for browser support is here, but if only support for recent browsers is required that could also be simplified using optional chaining (`?.` operator). [(Support Info)](https://caniuse.com/mdn-javascript_operators_optional_chaining)

Reply:
We're targeting ES6 minimum support, optional chaining is a tad too new, looks like it was part of ES2020. We should probably implement babel+uglify at some point to let us code with newer features while targeting ES6, but at the same time ES6 is pretty clean already